### ISSUE TITLE
[Bug]: Llama Index Deepinfra bug

### ISSUE THREAD

**Issue by krrishdholakia** [BerriAI/litellm#462]
### What happened?

Using llama index with litellm. When trying to call deepinfra, it throws an error.

**Comment by ghost** [comment]
https://github.com/shauryr/S2QA/blob/litellm/notebooks/litellm.ipynb
<img width="1016" alt="Screenshot 2023-09-27 at 9 10 26 AM" src="https://github.com/BerriAI/litellm/assets/17561003/48127334-717b-4a49-a088-d5b3795b08bb">

**Comment by ghost** [comment]
cc: @shauryr

**Comment by ishaan-jaff** [comment]
im taking a look at this 

I was able to fix this locally

**Comment by ishaan-jaff** [comment]
here's my deep infra call 
<img width="1280" alt="Screenshot 2023-09-28 at 1 15 42 PM" src="https://github.com/BerriAI/litellm/assets/29436595/5d168f7c-d514-401d-9561-d76f890c91ae">

TLDR: this is because our llama index implementation was trying to make text_completion requests for models like deep_infra

**Comment by ishaan-jaff** [comment]
@shauryr do you use llama index with streaming ?

**Comment by shauryr** [comment]
Yes, I use streaming

**Comment by ishaan-jaff** [comment]
I have a PR here it fixes regular calls for deep infra : https://github.com/jerryjliu/llama_index/pull/7885
